docs/ibc/light-clients/proofs.md

```diff
@@ -1,3 +1,24 @@
 <!--
 order: 7
--->
\ No newline at end of file
+-->
+
+# Existence and Non-Existence Proofs 
+
+IBC uses merkle proofs in order to verify the state of a remote counterparty state machine given a trusted root, and [ICS23](https://github.com/cosmos/ics23/tree/master/go) is a general approach for verifying merkle trees which is used in `ibc-go`.
+
+Currently, all Cosmos SDK modules contain their own stores, which maintain the state of the application module in an IAVL (immutable AVL) binary merkle tree format. Specifically with regard to IBC, core IBC maintains its own IAVL store, and IBC apps (e.g. transfer) maintain their own dedicated stores. The Cosmos SDK multistore therefore creates a simple merkle tree of all of these IAVL trees, and from each of these individual IAVL tree root hashes derives a root hash for the application state tree as a whole (the apphash).
+
+For the purposes of `ibc-go`, there are two types of proofs which are important: existence and non-existence proofs, terms which have been used interchangeably with membership and non-membership proofs. For the purposes of this guide, we will stick withe 'existence' and 'non-existence'. Existence proofs are used for transactions which will result in the writing of a packet receipt into the IBC store on the receiving end of the transaction (ie: token transfers, channel/connection handshakes), whereas non-existence proofs are used to timeout IBC packets.
```

Comment:
> Existence proofs are used for transactions which will result in the writing of a packet receipt into the IBC store on the receiving end of the transaction (ie: token transfers, channel/connection handshakes), whereas non-existence proofs are used to timeout IBC packets.

I think this sentence is a little inaccurate because no packet receipts would be written for channel/connection handshakes. What about something like below, could paraphrase it however you want! 

> Existence proofs are used in IBC transactions which involve verification of counterparty state. For example, this includes verification of connection and channel state within IBC protocol handshakes as well as proving the existence of packet commitments and receipts leveraged by IBC applications. Non-existence proofs verify the absence of data stored within counterparty state. For example, non-existence proofs are used to timeout IBC packets by verifying the absence of a packet receipt on a particular counterparty.

Reply:
Like @damiannolan 's suggestion, but why not add this to the appropriate sections below where we discuss existence and non-existence proofs?

Like so,
> For the purposes... we will stick with 'existence' and 'non-existence'.
>## Existence Proofs
>- explanation
>- use case (channel/conn handshake, packet receipt)
>## Non-existence Proofs
>- explanation
>- note about "mock"
>- use case of timeouts